frontend/src/app-loader.tsx

```diff
@@ -23,17 +19,18 @@ export function AppLoader({ children }: { children: React.ReactNode }) {
 
   const {
     state: { status, network, networkConfig },
+    actions,
     dispatch
   } = useGlobal()
 
   // Get wallets, service state and version
   useEffect(() => {
-    dispatch(initAppAction())
+    dispatch(actions.initAppAction())
   }, [dispatch])
```

Comment:
chore (blocking): We should add actions to the deps array here or an eslint-ignore if we definitely dont want to. Same for actions.startServiceAction too

Reply:
i think i'll try to add a linter to this repo, as it's super weird to have lint-like requests on PRs by humans :D the actions object never changes after definition, so i think the effect is going to be fine without it in this case.

Reply:
fixed!